packages/cli/templates/element/polymer-3.x/demo/index.html

```diff
@@ -8,8 +8,11 @@
 
     <script src="../node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
 
-    <script type="module" src="../node_modules/@polymer/iron-demo-helpers/demo-pages-shared-styles.js"></script>
-    <script type="module" src="../node_modules/@polymer/iron-demo-helpers/demo-snippet.js"></script>
+    <script type="module">
+      import '@polymer/iron-demo-helpers/demo-pages-shared-styles';
```

Comment:
`.js`

Reply:
The .js isn't actually needed, node resolution includes adding a .js. Have we typically been including them anyway?

Reply:
I think so (modulizer doesn't remove them, fwiw) but it doesn't seem like we're totally consistent since some of our projects make use of node's `main` field: https://github.com/Polymer/pwa-starter-kit/blob/8da3dcc42472e99fa22d0a996362828d31dc70bb/src/components/my-app.js#L11-L13

Reply:
IMO, we shouldn't have allowed these kinds of special behaviors from Node.js' specifier resolution in polyserve's rewriting and really hope this gets dropped in spec land.